Food/nim.lua

```diff
@@ -0,0 +1,175 @@
+local name = "nim"
+local version = "1.6.2"
+
+food = {
+    name = name,
+    description = "Nim is a statically typed compiled systems programming language. It combines successful concepts from mature languages like Python, Ada and Modula.",
+    homepage = "https://nim-lang.org",
+    version = version,
+    packages = {
+        {
+            os = "darwin",
+            arch = "amd64",
+            url = "https://github.com/DHunte/NimBuilds/releases/download/"  .. version .. "/" .. name .. "-darwin-" .. version .. "-x64.zip",
```

Comment:
Is this repository endorsed by the nim maintainers? Might be worth checking in…

Reply:
Hi, thanks for the advice. I opened a feature request on in their suggested repository and hopefully now waiting for a response. If there is no response I will close this pull-request as they currently aren't responding on discord to the question so its unclear.